int64 pesan1
float64 pesan2
string pesan3